sdk/helper/template/funcs.go

```diff
@@ -10,12 +10,16 @@ import (
 	UUID "github.com/hashicorp/go-uuid"
 )
 
-func nowSeconds() string {
+func unixTime() string {
 	return strconv.FormatInt(time.Now().Unix(), 10)
 }
 
-func nowNano() string {
-	return strconv.FormatInt(time.Now().UnixNano(), 10)
+func unixTimeMillis() string {
+	return strconv.FormatInt(time.Now().UnixNano()/int64(time.Millisecond), 10)
```

Comment:
Is there any reason to not go with nanoseconds? We should provide a way to either go with somewhat precise time (second) or very precise time (nanoseconds), similar to `Unix` and `UnixNano`.

Reply:
There isn't a need to provide that level of precision, especially if that level of precision adds additional characters to a username. The majority of systems out there that I've personally seen do not go down to the nanosecond level. The main reason I am even including milliseconds is to improve the ability to differentiate between usernames in systems that receive a lot of dynamic user requests. If nanoseconds is something that is desired we can add it at a later time.